docs/en/Running-Alluxio-on-GCE.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: global
+title: Running Alluxio on GCE
+nickname: Alluxio on GCE
+group: User Guide
+priority: 4
+---
+
+Alluxio can be deployed on Google Compute Engine (GCE) using the
+[Vagrant scripts](https://github.com/alluxio/alluxio/tree/master/deploy/vagrant) that come with
+Alluxio. The scripts let you create, configure, and destroy clusters.
+
+# Prerequisites
+
+**Install Vagrant and the Google plugin**
+
+Download [Vagrant](https://www.vagrantup.com/downloads.html)
+
+Install Google Vagrant plugin:
+
+{% include Running-Alluxio-on-GCE/install-google-vagrant-plugin.md %}
+
+**Install Alluxio**
+
+Download Alluxio to your local machine, and unzip it:
+
+{% include Common-Commands/download-alluxio.md %}
+
+**Install python library dependencies**
+
+Install [python>=2.7](https://www.python.org/), not python3.
+
+Under `deploy/vagrant` directory in your home directory, run:
+
+{% include Running-Alluxio-on-GCE/install-vagrant.md %}
+
+Alternatively, you can manually install [pip](https://pip.pypa.io/en/latest/installing/), and then
+in `deploy/vagrant` run:
+
+{% include Running-Alluxio-on-GCE/install-pip.md %}
+
+# Launch a Cluster
+
+To run an Alluxio cluster on GCE, first sign up for an Google Cloud account and create a project
+on the [Google Cloud Platform site](http://cloud.google.com/).
+
+If you are not familiar with Google Compute Engine, you can read the [documentation](http://cloud.google.com/compute/docs) first.
+
+Then create a [service account](https://console.google.com/)
+
+Next, generate a Google Cloud service account at the 
+[Google Cloud Console](http://console.cloud.google.com) Permissions page. In the dialog box for new Service Accounts, check "Furnish a new private key." Download and store the JSON key in a reliable location.
```

Comment:
In my Service Accounts page, there is already "Compute Engine default service account". Can I use this instead of creating new a Service Account? If so, can you please also document the steps by using existing Service Account?

Thanks!